pkg/puller/metrics.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package puller
+
+import (
+	m "github.com/ethersphere/bee/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type metrics struct {
+	HistWorkerIterCounter prometheus.Counter // counts the number of historical syncing iterations
+	HistWorkerDoneCounter prometheus.Counter // count number of finished historical syncing jobs
```

Comment:
afaik this one is never used. i'd suggest removing it or increasing it when the job is completed without errors. 

if we go for the 2nd option, i'd suggest adding a `LiveWorkerDoneCounter` as well.

Reply:
@mortelli `live` syncing should never be done, hence the asymmetry. nice catch that it wasn't used though. added it now :+1: